metaflow/plugins/package/package_decorator.py

```diff
@@ -0,0 +1,70 @@
+import os
+import sys
+import threading
+import time
+
+from metaflow.decorators import StepDecorator
+from metaflow.exception import MetaflowException
+from metaflow.package import MetaflowPackage
+
+
+class PackageDecorator(StepDecorator):
+    name = "package"
+
+    def set_remote_step(self, decorators):
```

Comment:
We will still need something like this. Maybe the "global decorator state" is a good palce to start with this.